client/src/components/tags/NewTag/NewTag.tsx

```diff
@@ -39,12 +39,12 @@ function NewTag({ modalId }: NewTagProps) {
 					</F.Label>
 				</F.Row>
 				<F.Row>
-					{Object.keys(tags?.byId ?? {}).length > 0 && (
+					{Object.keys(tagsData?.byId ?? {}).length > 0 && (
```

Comment:
Object.keys will appear to work, but will always be empty! Use proper method for Map.